config.json

```diff
@@ -61,6 +61,13 @@
       ]
     },
     {
+      "slug": "acronym",
+      "difficulty": 2,
+      "topics": [
+        "String Manipulation"
```

Comment:
if this one gets String Manipulation, should any other exercise that deals with String get this topic too?

For solving #274 I only included topics that will affect ordering decisions.

Do you propose that string manipulation affects ordering decisions, and if so can you help find the other exercises that have it so they can be labeled properly too? If a topic appears in this file, it is advantageous for it to appear on all exercises that have that topic, otherwise we don't get the full picture when making ordering decisions.